wpengine/faustjs
# chore: bump lerna to 4.0.0

To ensure that the `ssri` transitive dependency is using the security patch from v8.0.1.

### To test
- `npm run bootstrap` and `npm run dev` should still work from the project root.

The e2e test fail in CI is unrelated.
